weblate_web/templates/snippets/service.html

```diff
@@ -1,107 +1,167 @@
 {% load i18n %}
+{% load static %}
+{% load date_diff %}
 
-            <div class="payment-form">
-                <div class="form-line">
-                    <div class="line-left">{% trans "Service" %}</div>
+<div class="payment-form pt-pb">
+    <div class="form-line style6 bdr-btm">
+        <div class="service-variant">
+            <p>{{ service.get_status_display }}</p>
+        <div class="service-time">
+            {% with subscriptions=service.subscription_set.all %}
+            {% for subscription in subscriptions %}
+            {% if subscription.expires|days_diff_from_today < 30 %}
+            {% if subscription.expires|days_diff_from_today < 2 and subscription.expires|days_diff_from_today >= 0 %}
+            <t class="orange">{{ subscription.expires|days_diff_from_today}} day remaining</t>
+            {% elif subscription.expires|days_diff_from_today < 0  %}
+            <t class="orange">Expired</t>
+            {% else %}
+            <t class="orange">{{ subscription.expires|days_diff_from_today}} days remaining</t>
+            {% endif %}
+            {% else %}
+            <t class="grey">{{ subscription.expires|days_diff_from_today}} days remaining</t>
```

Comment:
Please add support for localization and use properly plurals here, see https://docs.djangoproject.com/en/3.1/topics/i18n/translation/#blocktranslate-template-tag (look for pluralization).

Reply:
It's done, please check!